docs/source/cognite.rst

```diff
@@ -2,13 +2,28 @@ Quickstart
 ==========
 Authenticate
 ------------
-The preferred way to authenticate against the Cognite API is by setting the :code:`COGNITE_API_KEY` environment variable. All examples in this documentation require that the variable has been set.
+
```

Comment:
A CDF project can be setup to either authenticate with OpenID Connect (OIDC) or API keys. OIDC is preferred and all new project will be setup with that. Projects setup before May 2021 will most likely use API keys unless it has been migrated to OIDC.   (it's not something the user can decide himself)